### PR TITLE
refactor layout files using Android suggested hints

### PR DESCRIPTION
**Pull request by connorreinhold**
## Overview

Standard refactoring of all xml files to address Android autofilled warnings (I.e. using left/right instead of start/end, having hardcoded strings)

## Changes Made

In addition to all the standard changes of layout, I put finish() in endOnboarding instead of starting a new intent, which was messing up the backstack after loading into MainActivity

## Test Coverage

Tested all major features, look the same as before. No display changes have occurred. 

## Next Steps

Refactor all the java files to remove potential NullPointers and add strings to assets

## Related PRs or Issues

#187 (unfinished)

### CODE REVIEW

File: app/src/main/res/layout/fragment_main_list.xml [hunk 1]
```diff
@@ -1,452 +1,452 @@
...
     tools:context=".MainListFragment">
 
-    <RelativeLayout
+    <HorizontalScrollView
+        android:id="@+id/scroll"
         android:layout_width="match_parent"
-        android:layout_height="match_parent">
+        android:layout_height="wrap_content"
+        android:paddingLeft="3dp"
+        android:paddingRight="3dp"
```
Comment (jmf373) [inline app/src/main/res/layout/fragment_main_list.xml]: nit: `paddingStart`/`paddingEnd`

File: app/src/main/res/layout/fragment_main_list.xml [hunk 2]
```diff
@@ -1,452 +1,452 @@
...
-                    android:textColor="@color/blue"
-                    android:textSize="16sp"
-                    android:textStyle="normal" />
+            <Button
+                android:id="@+id/brb"
+                android:layout_width="wrap_content"
+                android:layout_height="40dp"
+                android:layout_marginLeft="8dp"
+                android:layout_marginTop="15dp"
+                android:layout_marginRight="8dp"
```
Comment (jmf373) [inline app/src/main/res/layout/fragment_main_list.xml]: nit: `paddingStart`/`paddingEnd`

File: app/src/main/res/layout/fragment_main_list.xml [hunk 3]
```diff
@@ -1,452 +1,452 @@
...
-                    android:textSize="16sp"
-                    android:textStyle="normal"
-                    android:visibility="gone" />
+            <Button
+                android:id="@+id/vietnamese"
+                android:layout_width="wrap_content"
+                android:layout_height="40dp"
+                android:layout_marginLeft="8dp"
+                android:layout_marginTop="15dp"
+                android:layout_marginRight="8dp"
```
Comment (jmf373) [inline app/src/main/res/layout/fragment_main_list.xml]: nit: `paddingStart`/`paddingEnd`

**Comment by ningning621** [comment]
1000 lines of changes 😱

**Comment by ningning621** [comment]
also please do a CTRL + L + Shift + Alt to lint everything

**Comment by connorreinhold** [comment]
Linted almost everything. Idk why running the linter messes up the designs for a few of the fragments so I left a few unlinted

**Pull request merged**